src/app/core/facets-manager/facets-manager.component.scss

```diff
@@ -110,4 +110,24 @@
     }
 }
 
+.row {
+    margin-right: 125px;
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: space-between;
+}
+
+.col {
+    flex-grow: 1;
+    min-width: 30%;
+    margin-right: 20px;
+    margin-bottom: 15px;
+}
```

Comment:
This needs to be either moved out of facets-manager, or triggered by an additional variable besides 'showAllFacets' that's not in substances-browse